pkg/db/src/__tests__/tests.ts

```diff
@@ -140,16 +140,26 @@ export const noteTransactionOperations: TestFunction = async (db) => {
 
 	// Update transaction should overwrite the existing transaction (and not confuse it with the same isbn, but different warehouse)
 	await note.updateTransaction({ isbn: '11111111', quantity: 3, warehouseId: wh1._id });
+	// Update transaction should not overwrite the existing warehouseId if it's empty
+	await note.updateTransaction({ isbn: '0123456789', quantity: 3 });
+
+	// Update transaction should overwrite the existing warehouseId
+	await note.updateTransaction({ isbn: '0123456789', quantity: 4, warehouseId: wh2._id });
```

Comment:
Can we  replace this line to, instead of updating transaction with isbn "012345678", update the transaction
```ts
{ isbn: "11111111", quantity: 10, warehouseId: versionId(wh2._id) }
```
to 
```ts
{isbn: "11111111", quantity: 10, warehouseId: "warehouse-3" }
```